LottoNumberCreator/LottoNumberCreator/ViewController.swift

```diff
@@ -51,14 +55,36 @@ class ViewController: UIViewController {
             }
         }.resume()
     }
+    
+    /// alamofire request 방법
+    func afLottoNumber(api: API, min: Int, max: Int, count: Int) {
+        var baseURL = "https://www.randomnumberapi.com/api/v1.0"
+        baseURL.append("\(api.rawValue)?min=\(min)&max=\(max)&count=\(count)")
+        
+        AF.request("\(baseURL)").responseDecodable(of: [Int].self) { response in
+            switch response.result {
+            case .success(let data):
+                let lottoNumbers: String = {
```

Comment:
고차함수 `.map()`와 string관련 메서드인 `.joined()` 를 활용해볼 수 있습니다.

```suggestion
                let lottoNumbers = data.map{ String($0) }.joined(separator: ".")
```

Reply:
@keenkim1202 수정 완료했습니다!